### PR TITLE
Add "ngt" label

### PR DESCRIPTION
**Pull request by fwyzard**
Add the `ngt` label, to support the `type ngt` bot command.

### CODE REVIEW

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
type ngt

(I'm just curious to see what happens)

**Comment by smuzaffar** [comment]
@fwyzard , what is `ngt` ? 
If it is only used for adding `ngt` label by explicitly using `type ngt` comment then please include it in https://github.com/cms-sw/cms-bot/blob/master/githublabels.py#L32 . If we want bot to automatically set this label based on changes set then it should go in https://github.com/cms-sw/cms-bot/blob/master/categories_map.py#L1801

**Comment by fwyzard** [comment]
> what is `ngt` ?

the NextGen Trigger project, that is developing additional L1T and HLT functionalities for Phase 2.

The idea is to be able to quickly point out which developments come from this project, since we have to give yearly reports on the developments to the foundations that funds the project. 

> If it is only used for adding `ngt` label by explicitly using `type ngt` comment then please include it in https://github.com/cms-sw/cms-bot/blob/master/githublabels.py#L32

Yes, that was the idea.
I'll make the changes, thanks for the correction.

File: githublabels.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def get_dpg_pog():
         "performance|improvements|performance-improvements",
         "mtype",
     ],
+    "ngt": [LABEL_COLORS["performance"], "ngt", "mtype"],
```
Comment (fwyzard) [inline githublabels.py:44]: @smuzaffar is this what you had in mind ?

Comment (smuzaffar) [reply githublabels.py:44]: yes.

**Comment by fwyzard** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**